README.md

```diff
@@ -201,13 +201,13 @@ the `tpm2_checkquote` utility in your path.
 ###### TPM 2.0 Resource Manager
 
 Note that it is recommended that you use the tpm2-abrmd resource manager
-(available at https://github.com/tpm2-software/tpm2-abrmd) as well instead of
-communicating directly with the TPM.  See README.md at that project for
-detailed instructions on how to build and install.
+(available at [tpm2-abrmd](https://github.com/tpm2-software/tpm2-abrmd)) as well instead of
```

Comment:
Same here: remote the parenthetical and just make "tpm2-abrmd" link to the repo.